src/pipx/commands/install.py

```diff
@@ -29,11 +29,11 @@ def install(
     include_dependencies: bool,
     preinstall_packages: Optional[List[str]],
     suffix: str = "",
+    python_flag_passed: Optional[bool] = False
```

Comment:
`install` is being called from a few other places as well. Should we evaluate `python_flag_was_passed` everywhere it is being invoked and pass it down ?

Reply:
But even if `install` is being called without `python_flag_passed`, the default will set it to `False`, so the `Optional` type is unnecessary. `python_flag_passed` will always be a boolean, never `None`

Reply:
And preferably yes, we should always pass the correct `python_flag_passed` value.

Reply:
Done. Inferring the flag from the other place is just checking if `python` param was passed to the method or not. So I think `python_flag_passed=bool(python)` will work.

Reply:
It will always be passed, due to these lines, so inferring it the way you do is not accurate:
https://github.com/pypa/pipx/blob/6d81069806745ce29b1e85d016463bbee206380f/src/pipx/main.py#L203-L210

Reply:
Oh yeah, my bad. Then i guess we have to drill down the value of ‘python_was_passed’ to the point where install is used